app/controllers/v0/facilities/va_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class V0::Facilities::VaController < FacilitiesController
+  before_action :validate_params, only: [:index]
+
+  # Index supports the following query parameters:
+  # @param bbox - Bounding box in form "xmin,ymin,xmax,ymax" in Lat/Long coordinates
+  # @param type - Optional facility type, values = all (default), health, benefits, cemetery
+  # @param services - Optional specialty services filter
+  def index
+    results = VAHealthFacility.query(bbox: params[:bbox], services: params[:services])
+    render json: results,
+           serializer: CollectionSerializer,
+           each_serializer: VAHealthFacilitySerializer
+  end
+
+  def show
+    results = VAHealthFacility.find_by_id(id: params[:id])
+    raise Common::Exceptions::RecordNotFound, params[:id] if results.nil?
+    render json: results, serializer: VAHealthFacilitySerializer
+  end
+
+  private
+
+  def validate_params
+    bbp = params[:bbox].to_s.split(',')
```

Comment:
Even though this construct for array comes from the concept of a checklist, when working with query strings you are essentially passing form-url-encoded data. Those checklists that have the same name, are lumped together as the same result set. So I would say yes, it should follow the same paradigm if it is done as query strings provided by search because the concepts apply.

Only if you're doing a POST request with JSON payload, would passing an actual array be a good alternative. 

Passing a string that is comma separated is just bad in both instances.